bin/petemaild

```diff
@@ -2538,16 +2551,18 @@ if ($help) {
     exit(0);
 }
 
-mySociety::SystemMisc::log_to_stderr(0) if (!$debug);
-# XXX: We call daemon() here AND call it again a few lines later??
-mySociety::SystemMisc::daemon() unless ($debug);
-
 $SIG{TERM} = $SIG{INT} = sub { $foad = 1; };
 
+# Only daemonize if `--debug` or `--foreground` aren't given
 if ($debug) {
     Log::verbose($verbose);
```

Comment:
Log::verbose($verbose) is called by all branches of this if, looks like.

Reply:
Quite right, I've put this above the block in both.